plugin/src/main/resources/data/lang_config.properties

```diff
@@ -1,5 +1,6 @@
 en.rules.enabled=["CAN_NOT", "ARTICLE_MISSING", "ARTICLE_UNNECESSARY", "COMMA_BEFORE_AND", "COMMA_WHICH", "USELESS_THAT", "AND_ALSO", "And", "PASSIVE_VOICE"]
 en.rules.disabled=["WORD_CONTAINS_UNDERSCORE", "EN_QUOTES", "COMMA_PARENTHESIS_WHITESPACE", "LICENCE_LICENSE_NOUN_PLURAL", "LICENCE_LICENSE_NOUN_SINGULAR", "COMMA_PARENTHESIS_WHITESPACE", "MULTIPLICATION_SIGN", "DASH_RULE", "ARROWS", "SENTENCE_WHITESPACE", "UPPERCASE_SENTENCE_START", "EN_UNPAIRED_BRACKETS"]
 en.categories.disabled=["FALSE_FRIENDS"]
+de.categories.disabled=["TYPOGRAPHY"]
```

Comment:
Huh? :)

Reply:
There is nothing useful in this category, only grumbles at brackets ""